app/src/enclave_u/mod.rs

```diff
@@ -57,5 +61,6 @@ pub fn check_transfertx(
             txins_enc.len(),
         )
     };
+    println!("fee: {}", actual_fee_paid);
```

Comment:
I don't know if `info!()` or `debug!()` macros will work here or not. But, if they work, I'd suggest to replace this with log statements.

Reply:
oh, this one should be removed -- `env_logger` should work here